src/neuroglancer/util/color.ts

```diff
@@ -93,8 +93,8 @@ export function unpackRGBA(value: number) {
       ((value >>> 24) & 0xff) / 255);
 }
 
-export function serializeColor(x: vec3|vec4) {
-  if (x[3] === undefined || x[3] === 1) {
+export function serializeColor(x: vec3|vec4, ignoreAlpha=false) {
```

Comment:
added ignore alpha because that is essentially what is done in `getCSSColor`, I'm not sure why the color originally has a alpha of 0. I also considered making `getCssColor` optionally return the hex representation.

Reply:
Instead of ignoreAlpha you can just pass a vec3, e.g. color.subarray(0, 3) as vec3